api-specificatie/resolved/openapi.yaml

```diff
@@ -3761,20 +3806,33 @@ components:
             woonplaats heeft plaatsgevonden, binnen een gemeente. Alle karakters uit
             de MES-1 karakterset zijn toegstaan.
           example: AV-2019-08-19-00
+        ligtInWoonplaats:
+          $ref: '#/components/schemas/NEN3610ID'
+        mogelijkOnjuist:
+          $ref: '#/components/schemas/OpenbareRuimteMogelijkOnjuist'
+      description: Een Openbare ruimte is een door het bevoegde gemeentelijke orgaan
+        als zodanig aangewezen en van een naam voorziene buitenruimte die binnen één
+        woonplaats is gelegen. Deze definitie is herleidbaar naar maar niet conform
+        de BAG Catalogus 2018.
+    OpenbareRuimteMogelijkOnjuist:
+      type: object
+      properties:
         naamMogelijkOnjuist:
           type: boolean
+          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
+            juistheid van de waarde in het corresponderende gegeven.
         typeMogelijkOnjuist:
           type: boolean
+          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
+            juistheid van de waarde in het corresponderende gegeven.
         statusMogelijkOnjuist:
           type: boolean
-        gerelateerdeWoonplaatsMogelijkOnjuist:
+          description: Indicator die aangeeft of er onderzoek wordt gedaan naar de
+            juistheid van de waarde in het corresponderende gegeven.
+        mogelijkVerkeerdeWoonplaatsGebruikt:
```

Comment:
is dit echt duidelijker dan gewoon de propertynaam te gebruiken (ligtInWoonplaats)

Reply:
Nee dus. Heb ik weer aangepast.